### ISSUE TITLE
Kernel Shape of MaxPool2d not showing

### ISSUE THREAD

**Issue by martinandrovich** [TylerYep/torchinfo#112]
**Describe the bug**
When printing model summary with `"kernel_size"` column enabled, the `MaxPool2d` is not shown. Not sure if this is intentional.

**To Reproduce**
Something like this:
```py

# convolution block within a model
self.conv1 = nn.Sequential(
	nn.Conv2d(in_channels=1, out_channels=2, kernel_size=5, stride=1, padding=1),
	nn.MaxPool2d(kernel_size=2, stride=2),
	nn.ReLU(),
	nn.BatchNorm2d(2)
)
```
```py
# model summary
model_summary = summary(model, input_size=model.batch_shape, col_names=["kernel_size", "input_size", "output_size"], row_settings=["ascii_only"], col_width=20, verbose=0)
print(model_summary)
```

The full code is [available on my GitHub](https://github.com/martinandrovich/speaker_recognition).

**Output**
```
====================================================================================================
Layer (type)                             Kernel Shape         Input Shape          Output Shape
====================================================================================================
SpeechRecognitionNet                     --                   --                   --
+ Sequential                             --                   [24, 1, 64, 53]      [24, 2, 31, 25]
|    + Conv2d                            [1, 2, 5, 5]         [24, 1, 64, 53]      [24, 2, 62, 51]
|    + MaxPool2d                         --                   [24, 2, 62, 51]      [24, 2, 31, 25]
|    + ReLU                              --                   [24, 2, 31, 25]      [24, 2, 31, 25]
|    + BatchNorm2d                       [2]                  [24, 2, 31, 25]      [24, 2, 31, 25]
...
```

I would expect a Kernel Shape of `[2, 2, 2]` for a `2x2` kernel with stride `2` (or something alike).

**Context**
- Windows 11
- `Python 3.9.10`
- `torch==1.10.2+cu113`
- `torchinfo==1.6.3`

**Comment by TylerYep** [comment]
Good observation. Let me investigate and see if there's a better way of figuring out the kernel_size of layers.

**Comment by TylerYep** [comment]
This should be fixed in 793c4f57 and will be in v1.6.6
